src/ratings-and-reviews/index.jsx

```diff
@@ -18,25 +18,107 @@ const reviewsTotal = (reviews) => {
   return total;
 };
 
-const RatingsAndReviews = (/* { reviews, metadata } */) => (
-  <div id="ratings-and-reviews">
-    <div className="column-1">
-      <h2 className="reviews-header">Ratings & Reviews</h2>
-      <span className="reviews-rating">{metadata.rating.toFixed(1)}</span>
-      <div>
-        <StarRating rating={metadata.rating}/>
+/** @param {Object} reviews*/
+const reviewsAverage = (reviews) => {
+  let total = 0;
+  let count = 0;
+  for (let i in reviews) {
+    total += i * reviews[i];
+    count += reviews[i];
+  }
+  let average = (total / count).toFixed(1);
+  return Number.parseFloat(average);
+};
+
+let averageRating = reviewsAverage(metadata.ratings);
+
+/** @param {string} newSortType*/
+
+const sortReviews = (newSortType) => {
+  if (newSortType === 'Relevance') {
+
+    filteredReviews.sort(function (review1, review2) {
```

Comment:
You must have missed my message to you in the Slack. Relevance isn't something you're supposed to calculate on your own. [The API server does it for you.](https://learn-2.galvanize.com/cohorts/2967/blocks/94/content_files/Front End Capstone/project-atelier-catwalk/reviews.md) In practice, assume that if you don't sort the reviews by anything, they're already sorted by relevancy.

Reply:
I did see your post. The intention is to replace this function with the proper API calls, I had already written it before I saw your message.